### PR TITLE
Added Inbox and WatchList classes

### PR DESCRIPTION
**Pull request by Senpai-paj**

### CODE REVIEW

File: src/main/java/org/example/marketplacebackend/model/Inbox.java [hunk 1]
```diff
@@ -1,5 +1,32 @@
...
+@Entity
+@Table
 public class Inbox {
 
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private UUID id;
+
+  @OneToOne
+  private Account account;
```
Comment (Luxcorel) [inline src/main/java/org/example/marketplacebackend/model/Inbox.java:26]: Change these lines to the following:
```
@ManyToOne
private Account receiver;
```

**Review by Luxcorel — CHANGES_REQUESTED** [review]
After changing the two commented lines, this PR is approved.

File: src/main/java/org/example/marketplacebackend/model/Inbox.java [hunk 2]
```diff
@@ -1,5 +1,33 @@
...
+@Entity
+@Table
 public class Inbox {
 
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private UUID id;
+
+  @ManyToOne
+  private Account account;
```
Comment (Luxcorel) [inline src/main/java/org/example/marketplacebackend/model/Inbox.java:27]: This line should still be: `private Account receiver;`

**Review by Luxcorel — CHANGES_REQUESTED** [review]
Almost everything is correct now, except one line that isn't fixed yet. See code comment for quick fix.

**Comment by JakobEdvardsson** [comment]
LGTM

**Pull request merged**